test/jsdom/encoding.js

```diff
@@ -0,0 +1,464 @@
+"use strict";
+
+const assert = require("chai").assert;
+const describe = require("mocha-sugar-free").describe;
+const specify = require("mocha-sugar-free").specify;
+const before = require("mocha-sugar-free").before;
+const after = require("mocha-sugar-free").after;
+
+const fs = require("fs");
+const path = require("path");
+const http = require("http");
+const portfinder = require("portfinder");
+const jsdom = require("../..");
+
+describe("jsdom/encoding", { skipIfBrowser: true }, () => {
+  let server;
+  let testHost;
+
+  before(() => {
+    return new Promise((done, rej) => {
+      portfinder.getPort((err, port) => {
+        if (err) {
+          return rej(err);
+        }
+
+        server = http.createServer((req, res) => {
```

Comment:
Right, I was thinking about the ones with no encoding in the header.

If st changes to add encoding to the header we can override it. I left a note in https://github.com/isaacs/st/pull/53#issuecomment-210682912